conda-forge.yml

```
@@ -1,10 +1,10 @@
 build_platform:
-  linux_aarch64: linux_64
-  linux_ppc64le: linux_64
-  osx_arm64: osx_64
+  linux_ppc64le: linux_ppc64le  # tests fail when emulating power from x86
+  osx_arm64: osx_64  # cross compile due to cloud.drone.io outage
```

Drone isn't used for macOS ARM. Only Linux ARM, but as noted above it doesn't work there and we switch to emulated by default currently. So this comment can be dropped.

```suggestion
  osx_arm64: osx_64
```

---

OK. I misunderstood that other Issue. I guess there is no native osx_arm64 build environment right now.